test/check-application

```diff
@@ -27,37 +27,111 @@ class TestApplication(testlib.MachineCase):
 
         # show image listing toggle
         b.wait_present('#containers-images tr:contains("busybox:latest")')
-        b.wait_visible('#containers-images tr:contains("busybox:latest")')
         b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-images tbody tr:contains("busybox:latest") + tr button.btn-delete')
 
         # run a container (will exit immediately)
-        m.execute("podman run -d --name swamped-crate busybox sh")
+        m.execute("podman run -d --name test-sh alpine sh")
+        # run a container
+        m.execute("podman run -d --name swamped-crate busybox sleep 1000")
 
         # FIXME: UI needs to listen to change signals and update automatically
+
         b.reload()
-        b.wait_present("#content")
         b.enter_page("/podman")
         b.wait_present("#containers-containers-filter button")
 
-        # show all containers to interact with stopped ones
+        # show all containers and check status
         b.click("#containers-containers-filter button")
-        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.wait_present("#containers-containers-filter .dropdown-menu")
         b.click("#containers-containers-filter li[data-data='all'] a")
         b.wait_present("#containers-containers")
         b.wait_present('#containers-containers tr:contains("busybox:latest")')
-        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest")')
+        # check busybox is running and alpine is exited
+        b.wait_present('#containers-containers tr:contains("busybox:latest") td:contains("running")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest") td:contains("exited")')
 
         # show container listing toggle
         b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        # test the container stop dropdown menu under the listing toggle
+        self.containerStopDropdown()
 
+        #show running container
         b.click("#containers-containers-filter button")
         b.wait_present("#containers-containers-filter .dropdown-menu")
         b.click("#containers-containers-filter li[data-data='running'] a")
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_present('#containers-containers tr:contains("swamped-crate")')
+        b.wait_present('#containers-containers tr:contains("busybox:latest") + tr:contains("sleep 1000")')
```

Comment:
This should also have a `wait_not_present` for the stopped alpine container.

Reply:
Put test for the stopped alpine container in [#6](https://github.com/cockpit-project/cockpit-podman/issues/6)